core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -68,6 +66,11 @@ public static String getStageDir(String tablePath) {
     return getMetadataPath(tablePath) + CarbonCommonConstants.FILE_SEPARATOR + STAGE_DIR;
   }
 
+  public static String getStageSnapshotFile(String tablePath) {
+    return CarbonTablePath.getStageDir(tablePath) + CarbonCommonConstants.FILE_SEPARATOR +
+        "snapshot";
```

Comment:
"snapshot" should be defined as constant.

Reply:
fixed